docs/configuration-parameters.md

```diff
@@ -34,7 +34,7 @@ This section provides the details of all the configurations required for the Car
 | carbon.storelocation | spark.sql.warehouse.dir property value | Location where CarbonData will create the store, and write the data in its custom format. If not specified,the path defaults to spark.sql.warehouse.dir property. **NOTE:** Store location should be in HDFS or S3. |
```

Comment:
This property is deprecated in 2.0, user should use only spark.sql.warehouse.dir.
a) I think we should remove it
b) But where do we mention about the work around for upgrade from 1.6 to 2.0
@kunal642 , @QiangCai : please check and give your suggestions for both these points

Reply:
I have not removed this property. I added in NOTE section about deprecated in carbondata 2.0 version.

Reply:
remove it. They should not configure in 2.0 version

Reply:
done